Update 03-prepare-user-data.md (#1261)
spectrocloud/librarium: content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md

You may encounter the following scenarios when creating an Edge Installer configuration user data file. Use these examples as a starting point to help you create user data configurations that fit your needs. 

## Sample User Data
```yaml
stylus:
  site:

>     # host for hubble api to register device.

Reviewer: ```suggestion
    # The Palette API endpoint to use. The default value is api.spectrocloud.com.
```

    paletteEndpoint: api.spectrocloud.com

>     # edgeHostToken is auto-registration token when edge host registers with the saas by itself when it boots up
>     # this token can be generated by browsing to Tenant Settings -> Registration Token.
>     # specify name and select project id in which edge host should register itself

Reviewer: ```suggestion
    # The edgeHostToken is an auto-registration token to register the Edge host with Palette upon bootup.
    # This token can be generated by navigating to the Tenant Settings -> Registration Token.
    # Specify a name and select the project id with which the Edge host should register itself.
```

    edgeHostToken: aUAxxxxxxxxx0ChYCrO

>     # projectUid <Optional :need to provide if edge host registration token is not generated using project id>

Reviewer: ```suggestion
    # The Palette project ID the Edge host should pair with. This is an optional field if an edgeHostToken is used and the token was assigned to a project.
```

    projectUid: 12345677788

>     # tags which will be assigned to devices as labels

Reviewer: ```suggestion
    # Tags which that will be assigned to the device as labels.
```

    tags:
      key1: value1
      key2: value2
      key3: value3

>     # name of the device, this may also be referred to as the edge id or edge host id.  If no edge host name is specified
>     # one will be generated from the device serial number.  If stylus cannot the device serial number a random id will
>     # be used instead. In the case of hardware that does not have a serial number is highly recommended to specify the
>     # device name as a random name is not deterministic and may lead to a device being registered twice under different
>     # names.

Reviewer: ```suggestion
    # The device's name, may also be referred to as the Edge ID or Edge host ID.  If no Edge hostname is specified,
    # a hostname will be generated from the device serial number.  If the Edge Installer cannot identify the device serial number, then a random ID will
    # be generated and used instead. In the case of hardware that does not have a serial number, we recommended specifying a
    # random name, with minimal chances of being re-used by a different Edge host.
```
Language cleanup.

    name: edge-appliance-1

>     # Optional
>     # If edge host requires proxy to connect to palette instance or to pull images then do specify proxy information here

Reviewer: ```suggestion
    # Optional
    # If the Edge host requires a proxy to connect to Palette or to pull images, then specify the proxy information in this section
```

    network:
      # configures http_proxy
      httpProxy: http://proxy.example.com
      # configures https_proxy
      httpsProxy: https://proxy.example.com
      # configures no_proxy
      noProxy: 10.10.128.10,10.0.0.0/8

      # Optional: configures the global  nameserver for the system.
      nameserver: 1.1.1.1
      # configure interface specific info. If omitted all interfaces will default to dhcp
      interfaces:
           enp0s3:
               # type of network dhcp or static
               type: static
               # Ip address including the mask bits
               ipAddress: 10.0.10.25/24
               # Gateway for the static ip.
               gateway: 10.0.10.1
               # interface specific nameserver
               nameserver: 10.10.128.8
           enp0s4:
               type: dhcp
    caCerts:
      - |
        ------BEGIN CERTIFICATE------
        *****************************
        *****************************
        ------END CERTIFICATE------
      - |
        ------BEGIN CERTIFICATE------
        *****************************
        *****************************
        ------END CERTIFICATE------

> # there is no password specified to default kairos user. To access edge host console, we will need to specify authorized keys or password. 

Reviewer: ```suggestion
# There is no password specified to the default kairos user. You must specify authorized keys or passwords to access the Edge host console. 
```

  poweroff: true
```

> **Site** - supplied at the edge location through a bootable USB drive. `projectName` if specified, will override project information specified in stage user-data's edge host registration token. Optionally, additional tags can also be specified. If edge site requires proxy for outbound connection then it must be provided in site user-data.

Reviewer 2: ```suggestion
**Site** - supplied at the edge location through a bootable USB drive. If specified, the `projectName` value overrides project information specified in the user data's edge host registration token. Optionally, you can add tags `tags:city`, `tags:building`, and `tags:zip-code`.
```

# Next Steps

> The last step of the EdgeForce workflow is to build the Edge artifact. Check out the [Build Images](/clusters/edge/edgeforge-workflow/build-images) guide to learn how to build your images with the Edge Installer Builder CLI and create an Edge artifact.

Reviewer 2: I don't see a change here.
